readme.md

```diff
@@ -176,6 +177,7 @@ By default, the module already registers some default options (e.g. "version" an
 | help | Automatically render the usage information when running `help`, `-h` or `--help` | true | Boolean |
 | name | The name of your program to display in help | Name of script file | String |
 | version | Outputs the version tag of your package.json | true | Boolean |
+| author  | Automatically render author information when running `author` | true | Boolean |
```

Comment:
We need to list the "authors" alias here. In addition, the property is now called "authors", not "author".